analyzers/PapasHistoryPrinter.py

```diff
@@ -0,0 +1,49 @@
+from heppy.framework.analyzer import Analyzer
+from heppy.papas.pfalgo.historyhelper import HistoryHelper
+
+
+class PapasHistoryPrinter(Analyzer):
+    '''Produces experimental History outputs for a papasevent (plots or text summary)
```

Comment:
Yes there is graph analysis done twice. TODO /TODISCUSS caching of graph analysis.